### ISSUE TITLE
Docker tag scorecard-action:v2.0.5 not found 

### ISSUE THREAD

**Issue by konstruktoid** [ossf/scorecard-action#978]
Renovate PR: "Update ossf/scorecard-action action to v2.0.5" (https://github.com/konstruktoid/ansible-role-hardening/pull/185)

https://github.com/konstruktoid/ansible-role-hardening/actions/runs/3276257156/jobs/5393791624#step:2:1

```console
Pull down action image 'gcr.io/openssf/scorecard-action:v2.0.5'
  /usr/bin/docker pull gcr.io/openssf/scorecard-action:v[2](https://github.com/konstruktoid/ansible-role-hardening/actions/runs/3276257156/jobs/5393791624#step:2:2).0.5
  Error response from daemon: manifest for gcr.io/openssf/scorecard-action:v2.0.5 not found: manifest unknown: Failed to fetch "v2.0.5" from request "/v2/openssf/scorecard-action/manifests/v2.0.5".
  Warning: Docker pull failed with exit code 1, back off 5.34 seconds before retry.
  /usr/bin/docker pull gcr.io/openssf/scorecard-action:v2.0.5
  Error response from daemon: manifest for gcr.io/openssf/scorecard-action:v2.0.5 not found: manifest unknown: Failed to fetch "v2.0.5" from request "/v2/openssf/scorecard-action/manifests/v2.0.5".
  Warning: Docker pull failed with exit code 1, back off [3](https://github.com/konstruktoid/ansible-role-hardening/actions/runs/3276257156/jobs/5393791624#step:2:3).379 seconds before retry.
  /usr/bin/docker pull gcr.io/openssf/scorecard-action:v2.0.5
  Error response from daemon: manifest for gcr.io/openssf/scorecard-action:v2.0.5 not found: manifest unknown: Failed to fetch "v2.0.5" from request "/v2/openssf/scorecard-action/manifests/v2.0.5".
Error: Docker pull failed with exit code 1
```

**Comment by naveensrinivasan** [comment]
This PR broker this https://github.com/ossf/scorecard-action/pull/961/files.

Thanks, I'll get a PR out soon and a fix.

**Comment by naveensrinivasan** [comment]
Fixed v2.0.6
```
docker pull gcr.io/openssf/scorecard-action:v2.0.6
v2.0.6: Pulling from openssf/scorecard-action
bd159e379b3b: Pull complete
8bcf04acd839: Pull complete
29b14408ebd9: Pull complete
e908b0653632: Pull complete
Digest: sha256:0fed0b03eb8f36794f3aaa31a3f61dac3ab646da816d579fcd2ef531ef437f06
Status: Downloaded newer image for gcr.io/openssf/scorecard-action:v2.0.6
gcr.io/openssf/scorecard-action:v2.0.6
```
